Changes on several pages (#83)
DHP-stottepakke/pages: pages/dmp_knowledge_base/difficult_faq.md

Practices for DMP evaluation (first draft/as part of the final reporting) are still under establishment and routines may vary between different research funders and institutions.

In the Horizon Europe funding scheme, a brief outline of data management (1-page DMP) is required at proposal stage and included the scientific proposal evaluation.
* [Horizon Europe - briefing slides for evaluators](https://ec.europa.eu/info/funding-tenders/opportunities/docs/2021-2027/experts/standard-briefing-slides-for-experts_he_en.pdf)
* [Horizon Europe - video on evaluation of Open Science practices](https://ec.europa.eu/info/funding-tenders/opportunities/portal/screen/support/videos), [YouTube-link](https://www.youtube.com/watch?v=EiJ8RaD3WBw)

### Where can a DMP be published?

> * [Zenodo](https://zenodo.org/) is an agnostic research archive (operated by CERN) which allows you to share all sorts of research outputs including data managment plans. Zenodo also allows versioning and will create a DOI for your files.

Reviewer: ```suggestion
* [Zenodo](https://zenodo.org/) is an agnostic research archive (operated by CERN) which allows you to share all sorts of research outputs including data managment plans. Zenodo also allows versioning and will create a unique DOI for your DMP versions.
```

You should consider a format which is common to the research discipline you are working in, which also adheres to [Open Science standards.](https://opendatahandbook.org/guide/en/appendices/file-formats/)

### What metadata should I consider collecting in my reseach project?
If you are unsure about whuch metadata to collect, consider consulting the [The Dublin Core Metadata standard](https://www.dublincore.org/), which originally comprosed of a [collection of 15 "essential" terms](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#section-3).
If you choose to share your data in a long term archive, it is highly likely this archive has a set of guidelines, wherein the necessary metadata are described. 

### Where can I find an example of how a DMP should look like?
Reading through different DMPs can be an useful exercise. Keep in mind that not all public DMPs are quality-controlled.

> Reading through different DMPs can be an useful exercise. Keep in mind that not all public DMPs are quality-controlled.

Reviewer: ```suggestion

```

* [Example DMPs and guidance from Digital Curation Centre](https://www.dcc.ac.uk/resources/data-management-plans/guidance-examples)
* [Curated collection of Horizon 2020 DMPs from University of Vienna](https://phaidra.univie.ac.at/search#?page=1&pagesize=10&collection=o:1140797)
* [DMP catalogue from LIBER Europe](https://libereurope.eu/working-group/research-data-management/plans/)
* [Public DMPs from Argos OpenAIRE](https://argos.openaire.eu/explore-plans)

## Level of the DMP

### I need a 1-page DMP for a Horizon Europe application
#### [European Commission. (2021, updated 2023). Horizon Europe Programme Guide: Evaluation of open science practices p.42ff](https://ec.europa.eu/info/funding-tenders/opportunities/docs/2021-2027/horizon/guidance/programme-guide_horizon_en.pdf "2024-09-12")

> > * Proposers will have to provide concrete information on how they plan to comply with the mandatory open science practices. Failure to sufficiently address this, will result

Reviewer: ```suggestion
> * Proposers will have to provide concrete information on how they plan to comply with the mandatory open science practices. Failure to sufficiently address this, will result in a lower evaluation score. A clear explanation of how they will adopt recommended practices, as appropriate for their projects, will result in a higher evaluation score. [page 42]
```

> in a lower evaluation score. A clear explanation of how they will adopt recommended practices, as appropriate for their projects, will result in a higher evaluation score. [page 42]

Reviewer: ```suggestion
```
